src/Library/demos/Web View/main.js

```diff
@@ -0,0 +1,91 @@
+import WebKit from "gi://WebKit?version=6.0";
+const GObject = imports.gi.GObject;
+const { Uri } = imports.gi.GLib;
+
+const container = workbench.builder.get_object("container");
+const button_back = workbench.builder.get_object("button_back");
+const button_forward = workbench.builder.get_object("button_forward");
+const button_reload = workbench.builder.get_object("button_reload");
+const button_stop = workbench.builder.get_object("button_stop");
+const url_bar = workbench.builder.get_object("url_bar");
+const web_view = new WebKit.WebView({
+  zoom_level: 0.8,
+});
+container.child = web_view;
+
+// URL bar displays the current loaded page
+web_view.bind_property(
+  "uri",
+  url_bar.buffer,
+  "text",
+  GObject.BindingFlags.DEFAULT,
+);
+
+// URL bar displays the current loaded page
+web_view.load_uri("https://www.gnome.org/");
+
+url_bar.connect("activate", () => {
+  let url = url_bar.buffer.text;
+  const scheme = Uri.peek_scheme(url);
+  if (scheme == null) {
+    url = `https://${url}`;
```

Comment:
```suggestion
    url = `http://${url}`;
```

default for the web is still `http` - the website will switch to https if it is supported